Openshift-EE/pipelines/OpenEBS-base/stages/3-functional/8QQP-Jiva-Snapshot-Delete/jiva-snapshot-delete

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+set -x
+
+pod() {
+# sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'git clone -b OpenEBS-base https://github.com/openebs/e2e-openshift.git'
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'cd e2e-openshift && bash Openshift-EE/pipelines/OpenEBS-base/stages/3-functional/8QQP-Jiva-Snapshot-Delete/jiva-snapshot-delete node '"'$CI_JOB_ID'"'' '"'$CI_PIPELINE_ID'"' '"'$CI_COMMIT_SHA'"'
+}
+
+node() {
+job_id=$(echo $1)
+pipeline_id=$(echo $2)
+commit_id=$(echo $3)
+case_id=8QQP
+
+source ~/.profile
+gittoken=$(echo "$github_token")
+
+time="date"
+current_time=$(eval $time)
+
+present_dir=$(pwd)
+echo $present_dir
+
+bash Openshift-EE/utils/e2e-cr jobname:jiva-snapshot-delete jobphase:Waiting
+bash Openshift-EE/utils/e2e-cr jobname:jiva-snapshot-delete jobphase:Running init_time:"$current_time" jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
+
+################
+# LitmusBook 1 #
+################
+
+: << EOF
+  --------------------------------------------------------------------------------------------------------------------     
+ | specAttribute     | kind   |         default value               | test specifc value                              |
+  ------------------------------------------------------------------------------------------------------------------- |
+ | storage clas      | env    | openebs-jiva-standard               | openebs-jiva-default                            |
+ | appNamespace      | env    | fio                                 | fio-jiva-snapshot-del                           |
+  ---------------------------------------------------------------------------------------------------------------------
+EOF
+
+test_name=$(bash Openshift-EE/utils/generate_test_name testcase=litmus-di-fio-snap-del- metadata="")
```

Comment:
@shashank855 testcase should be the name obtained from test name in test_vars of litmusbook 
`https://github.com/litmuschaos/litmus/blob/master/experiments/functional/snapshot_deletion/jiva/test_vars.yml`